manifests/params.pp

```diff
@@ -29,9 +29,18 @@
   if $::operatingsystem == 'Ubuntu' {
     if versioncmp($::lsbdistrelease, '8.04') < 1 {
       $init_style = 'debian'
-    } else {
+    } elsif versioncmp($::operatingsystemrelease, '15.04') < 0 {
       $init_style = 'upstart'
     }
+    else {
```

Comment:
To get a mixture while you guys are transitioning why not to do something like below?

```
$init_type = $::is_systemd ? {
  true    => 'systemd',
  default => 'upstart',
}

class { '::consul':
  init_type => $init_type,
}
```

I guess I don't understand why `is_systemd` should be a fact when we don't have `is_upstart`, `is_launchd`, `is_sysv`, etc. facts. So that makes me think we are headed the wrong direction with this :confused: